README.md

```diff
@@ -27,11 +27,7 @@ On your `.storybook/main.ts` file, add the following:
 
 ```js
 const config = {
-  ...
-  addons: [
-    ...
-    "storybook-addon-playground",
-  ],
+  addons: [..."storybook-addon-playground"],
```

Comment:
It looks like you need to spread this plugin, how about 
```
  addons: ["storybook-addon-playground"],
```

Reply:
Oh it's a mistake, the dots are for indicating the consumer "rest of your addons are here"